### PR TITLE
Update continuous-profiling.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lilyjazz** [comment]
@crazycs520 PTAL

File: dashboard/continuous-profiling.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
 
 - 该功能适用于使用 v1.9.0 及以上版本 TiUP 或 v1.3.0 及以上版本 TiDB Operator 部署和升级的集群，不支持二进制包部署和升级的集群。
 
+- 在 TiKV 和 TiFlash 实例中有偶发性失败，将会导致实例重启。
```
Comment (breeswish) [inline dashboard/continuous-profiling.md:24]: How about moving this into the "WARNING" section above?

**Review by breeswish — APPROVED** [review]

File: dashboard/continuous-profiling.md (hunk 1 above)
Comment (lilyjazz) [reply dashboard/continuous-profiling.md]: Done

**Review by qiancai — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Comment by breeswish** [comment]
@lilyjazz Should we hold the PR for now? Since Conprof will no longer profile TiFlash in v6.0.0 and the crash of TiKV has been fixed.

**Pull request closed without merging**